app/assets/stylesheets/wpcc/section/_contributions.scss

```diff
@@ -39,7 +39,7 @@
   .contributions__source-title {
     @include body(16, 22);
     color: $color-black;
-    margin: $baseline-unit*2 0 0 0;
+    margin: $baseline-unit*2 0;
```

Comment:
I would probably remove the margin-top on the <p> sibling element so that when the text shows it will still look aligned with the box on the right.